### ISSUE TITLE
Add support for spread syntax

### ISSUE THREAD

**Issue by leedm777** [power-assert-js/power-assert#108]
This is supported natively in Node.js v8 and v10.

h3. Steps to reproduce

See [GitHub repo](https://github.com/leedm777/power-spread) with demo

```shell
$ git clone https://github.com/leedm777/power-spread.git

$ cd power-spread

$ node --version
v8.11.1

$ npx mocha


  spread operator
    ✓ should work


  1 passing (6ms)


$ npx mocha --require intelli-espower-loader

/Users/dlee/prj/tmp/power-spread/node_modules/espower-source/index.js:165
    var err = new SyntaxError(detail);
              ^
SyntaxError:
     1: const assert = require('assert');
     2:
     3: describe('spread operator', () => {
     4:   it('should work', () => {
     5:     const obj = { a: 1, b: 2 };
     6:     const actual = { ...obj, c: 3 };
                            ^
     7:     assert.deepStrictEqual(actual, { a: 1, b: 2, c: 3 });
     8:   });
     9: });
    10:

Parse Error: Unexpected token (6:21) in /Users/dlee/prj/tmp/power-spread/test/spread.js
```

h3. Expected results

`mocha --require intelli-espower-loader` should parse and execute the tests.

h3. Actual results

Fails to parse the spread operator.

**Comment by twada** [comment]
Sorry for my late response. Will be fixed in forthcoming next minor release.

**Comment by chadxz** [comment]
I believe that will be 1.6.0

**Comment by twada** [comment]
Yep. Will be released soon as 1.6.0.

**Comment by twada** [comment]
[Just released as power-assert 1.6.0](https://github.com/power-assert-js/power-assert/blob/master/CHANGELOG.md#160-2018-06-14), thank you for reporting @leedm777 !
